index.html

```diff
@@ -1,75 +1,101 @@
-            <div class="signing-action">
-                <p>Already have an Intuit account?</p>
-                <a href="#">I forgot my user ID or password.</a>
-            </div>
-            <button type="button">Sign In</button>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title></title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <div class="sign-in-container">
+      <div class="sign-in-action">
+        <p>Already have an Intuit account?</p>
+        <a href="#">I forgot my user ID or password.</a>
+      </div>
+      <button type="button">
+          <span class='lock-icon'></span>
+          <p>Sign In</p>
+      </button>
+    </div>
+    <div class="main">
+      <div class="center-column">
+        <div class="logo-type"></div>
+        <div class="icons">
+          <div class="mint-icon">
+            <img
+              src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/mint-favicon.svg"
+              alt=""
+            />
+            <p>mint</p>
+          </div>
+          <div class="quickbooks-icon">
+            <img
+              src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/quickbooks-favicon@2x.svg"
+              alt=""
+            />
+            <p>quickbooks</p>
+          </div>
+          <div class="turbotax-icon">
+            <img
+              src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/tt-favicon@2x.svg"
+              alt=""
+            />
+            <p>turbotax</p>
+          </div>
         </div>
-        <div class="main">
-            <div class="center-column">
-                <div class="logo-type"></div>
-                <div class='icons'>
-                    <div class="mint-icon">
-                        <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/mint-favicon.svg" alt="">
-                        <p>mint</p>
-                    </div>
-                    <div class="quickbooks-icon">
-                        <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/quickbooks-favicon@2x.svg" alt="">
-                        <p>quickbooks</p>
-                    </div>
-                    <div class="turbotax-icon">
-                        <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/tt-favicon@2x.svg" alt="">
-                        <p>turbotax</p>
-                    </div>
+        <!--Container for the form in the middle of the page------------>
+        <div class="forms-container">
+          <section id="form-body">
+            <h2>Create an Intuit Account</h2>
+            <p class="subtitle">
+              One account for everything Intuit, including Mint.
+            </p>
+            <a href="#">Learn More</a>
+            <form class="sign-up-form" action="index.html" method="post">
+              <label for="e-mail">
```

Comment:
Let us use ```<label>``` tag as their name suggests. ie `<label for="email">email</label> <input type="email>` You may the wrap them inside one ```div``` or whichever means you need to make them align the way you want. I believe that would be more semantic than it is now.🙂
> use this approach in the other related instances